src/libmongoc/src/mongoc/mongoc-handshake.c

```diff
@@ -196,6 +196,10 @@ _mongoc_handshake_get_config_hex_string (void)
    _set_bit (bf, byte_count, MONGOC_MD_FLAG_ENABLE_MONGODB_AWS_AUTH);
 #endif
 
+   if (MONGOC_SRV_ENABLED) {
+      _set_bit (bf, byte_count, MONGOC_MD_FLAG_ENABLE_SRV);
```

Comment:
Confirming, with this change, we will now be including and tracking use of the C Driver with SRV enabled/disabled, correct?

Reply:
Correct